### ISSUE TITLE
show message when no state has been selected

### ISSUE THREAD

**Issue by jpyuda** [usagov/vote-gov#152]
If no state has been selected on the home page, we should show a message reminding the user to select their state to proceed.

**Comment by hallmedia** [comment]
As a general usability practice, the preferred method is to prevent the error in the first place. This can be handled many ways. I would recommend field legend copy that makes it clear the selection is required, and disable and/or change the default visual state of the button, then enable it once the selection is made / validated.

Further, even though there are many option values to present, consider an alternate UI to do so. Select fields are the least usable form element in history and create a great deal of friction for devices with touch input.

**Comment by jpyuda** [comment]
This is true, but in testing we did not encounter any users who didn't select a state. Adding copy that the field is required will simply add to the cognitive load of the form for the vast majority of users.

We also did some lightweight testing around different types of ways to select one's state -- on mobile, select fields performed the best, as they don't require typing. It also has the added benefit of allowing us to not require javascript and keep page weight low. We'll probably do some progressive enhancement of this page over time, but we'll see where the design on that lands.

Thanks for checking in!
